clients/rospy/src/rospy/topics.py

```diff
@@ -97,6 +97,7 @@ def isstring(s):
 from rospy.impl.tcpros_pubsub import QueuedConnection
 
 _logger = logging.getLogger('rospy.topics')
+_rosout_logger = logging.getLogger('rosout')
```

Comment:
I do not understand why it needs to change logger name. I believe that it should stick with "rospy.topics".

Reply:
It's not changing it, but referencing the `rosout` logger instance as an additional one.

The `rospy.topics` logger seems to be not logging visibly by default for normal users, so I added the rosout logger to make sure this warning ends up clearly visible for the user. It works, but as I said I don't know about the desired logger hierarchy in this package.

Reply:
IMO, since this message related to rospy.topics (Not rosout), i think it is better to be logger("rospy.topics").  if rospy.topics warning does not work, maybe we would make issue against it?

Reply:
Hmm... on second thought, I think this should actually be an exception rather than a warning. Because the program won't behave as expected by the user, so it won't be very useful to just warn.

>  if rospy.topics warning does not work, maybe we would make issue against it?

It works, but it seems to be unrelated to the standard rosout logger. This is fine if the `rospy.topics` logger is just intended for internal logging and not for end users.